rosdep/python.yaml

```diff
@@ -5463,6 +5463,10 @@ python3-cycler:
     '*': [python3-cycler]
     '7': null
   ubuntu: [python3-cycler]
+python3-dataclasses-json:
```

Comment:
So it looks like this is available on Ubuntu/Debian as a system package:

https://pkgs.org/search/?q=dataclasses-json

We prefer the system packages version (since things that depend on it can be released to the buildfarm), so please use the system packages for `arch`, `debian`, `ubuntu`, and `opensuse`.

Reply:
@clalancette Hi, I've update this for `debian`, `freebsd`, `opensuse` and `ubuntu`, but the package in `arch` is not official, should I use it?

Reply:
> @clalancette Hi, I've update this for `debian`, `freebsd`, `opensuse` and `ubuntu`, but the package in `arch` is not official, should I use it?

No, we only list official packages in rosdep.  In that case, this looks good, thanks for updating!